src/wp-admin/js/updates.js

```diff
@@ -305,6 +305,10 @@
 				$menuItem  = $( '#menu-appearance' );
 				$itemCount = $menuItem.find( '.theme-count' );
 				break;
+
+			default:
+				window.console.error( '"%s" is not white-listed to have its count decremented.', type );
```

Comment:
Attenzione signore. `wp.updates.decrementCount` is called by `wp.updates.updateItemSuccess`, which handles any type (translation updates need to be decremented too!). You should be able to pass all valid types to that function, but for 'plugin' and 'theme' it does some more stuff.